package-lock.json

```diff
@@ -1,12315 +0,0 @@
-{
```

Comment:
Thanks so much for your work on this, @pkfec! Did you mean to delete the package-lock file?

Reply:
I accidentally deleted the package-lock.json which is unrelated to django upgrade. I dropped this unnecessary commit. Thanks for pointing it out @lbeaufort